pyanaconda/payload/dnf/payload.py

```diff
@@ -89,9 +89,9 @@ def _set_default_source(self, opts):
         if self.proxy.Sources:
             log.debug("The DNF source is already set.")
 
-        elif opts.method:
+        elif opts.repo:
             log.debug("Use the DNF source from opts.")
-            source_proxy = self._create_source_from_url(opts.method)
```

Comment:
Hmm, I wonder about this. We should run payload kickstart tests to test different payloads are correctly created.

Reply:
I also found it weird.